src/main/java/org/gridsuite/modification/server/dto/byfilter/equipmentfield/BatteryField.java

```diff
@@ -64,7 +64,8 @@ public static void setNewValue(Battery battery, String batteryField, @NotNull St
                 ActivePowerControlAdder<Battery> activePowerControlAdder = battery.newExtension(ActivePowerControlAdder.class);
                 ModificationUtils.getInstance().modifyActivePowerControlAttributes(
                         activePowerControl, activePowerControlAdder, null,
-                        new AttributeModification<>(Float.parseFloat(newValue), OperationType.SET), null, null);
+                        new AttributeModification<>(Float.parseFloat(newValue), OperationType.SET), null,
+                    null, MODIFY_BATTERY_ERROR, "Battery '" + battery.getId() + "' : ");
```

Comment:
create const variable errorMessage for "Generator '" + generator.getId() + "' : "